Tasks/SqlAzureDacpacDeployment/Utility.ps1

```diff
@@ -250,3 +265,17 @@ function ConvertParamToSqlSupported
     return $param
 }
 
+function Test-CommandExists 
```

Comment:
Alternatively, you can write this also in single condition of if block as  if (Get-Command $commandName -ErrorAction SilentlyContinue) {}